src/main/resources/log4j2.xml

```diff
@@ -28,10 +28,17 @@
                 <SizeBasedTriggeringPolicy size="20 MB"/>
             </Policies>
         </RollingRandomAccessFile>
+
+        <File name="HTMLAppender"
+              fileName="${DEFAULT_HOME}/userContent/Log-${date:MM-dd-k-m}.html"
```

Comment:
Might want to name this something with the word "audit" in it. And the typical naming scheme would be camelCase, so a file name might be `auditLog-${date...}`